yroots/subdivision.py

```diff
@@ -630,7 +657,7 @@ def subdivision_solve_nd(funcs,a,b,deg,target_deg,interval_data,root_tracker,tol
         approx_errors.append(approx_error)
         # Subdivides if a bad approximation
         if coeff is None:
-            intervals = get_subintervals(a,b,np.arange(dim),interval_data,cheb_approx_list,change_sign,approx_errors,True)
+            intervals = get_subintervals(a,b,[i for i in range(dim)],interval_data,cheb_approx_list,change_sign,approx_errors)
```

Comment:
Maybe this is something we could memoize? It's very small but we use it over and over. Probably you'll have to test to see if it's worth memoizing

Reply:
That's a good suggestion! I'll try it and see if it gives any noticeable speedup. Regardless, I'm sure it will make things at least a little faster.